Sources/Rendering/Core/Actor/index.js

```diff
@@ -158,6 +158,12 @@ function vtkActor(publicAPI, model) {
 
```

Comment:
How about an example that prints out all the information you get by picking when moving the mouse around ?